crate/src/jepsen/crate.clj

```diff
@@ -122,44 +142,28 @@
               (.execute)
               (.actionGet)))))))
 
-;; Client
-(defn connect
-  "Opens a connection to a node."
+(def cratedb-spec {:dbtype "crate"
+                   :dbname "test"
+                   :classname "io.crate.client.jdbc.CrateDriver"
+                   :user "crate"
+                   :password ""
+                   :port 5432})
```

Comment:
Let's replace `5432` port with another one to avoid any conflicts from accidentaly running postgres or another CrateDB instance on the same machine.